x-pack/metricbeat/module/awsfargate/task_stats/container.go

```diff
@@ -15,6 +15,7 @@ type container struct {
 	Name     string
 	Image    string
 	Labels   map[string]string
+	Limits   Limits `json:"Limits"`
```

Comment:
nit: As the JSON key is the same as the struct field name, we can remove the tag `json:"Limits"` like other field names in the same struct